CMakeSettings.json

```diff
@@ -1,20 +1,38 @@
-{
+﻿{
   "configurations": [
     {
-      "name": "x64-Debug",
-      "generator": "Visual Studio 15 2017 Win64",
+      "name": "Debug",
+      "generator": "Visual Studio 16 2019 Win64",
       "configurationType": "Debug",
-      "buildRoot": "${env.USERPROFILE}\\CMakeBuilds\\${workspaceHash}\\build\\${name}",
+      "inheritEnvironments": [ "msvc_x64_x64" ],
+      "buildCommandArgs": "-m -p:PreferredToolArchitecture=x64",
+      "buildRoot": "${workspaceRoot}\\build",
```

Comment:
should this use build-debug instead?

Reply:
> should this use build-debug instead?
Debug builds on Windows build into the same folder under the name "DolphinD" rather than using a separate build folder so this won't cause any issues. The current implementation keeps that behavior.
https://github.com/spycrab/dolphin/blob/e3805b71f1d9ee879ba2bbbfd5fd2a5f563ee2ce/Source/Core/DolphinQt/CMakeLists.txt#L160-L164

Reply:
Yeah, what NarryG wrote.

Reply:
This seems to create problems with debug and release builds coexisting. One overwrites the other?

How about instead of copying the binaries to the build directory, we use the install step to copy everything into <root>/Binaries?

Reply:
It really shouldn't, as the debug binary is named DolphinD and the release one just Dolphin.

Reply:
I'm referring to the the intermediate/object files, not the resulting binary.

Reply:
Well it would probably act just as it would when you reconfigure your CMake project.